x11-misc/pcmanfm-qt/pcmanfm-qt-9999.ebuild

```diff
@@ -40,13 +39,6 @@ DEPEND="${CDEPEND}
 	sys-devel/gettext
 	virtual/pkgconfig"
 
-src_prepare() {
-	# fix multilib
-	sed -i -e "/LIBRARY\ DESTINATION/s:lib:$(get_libdir):" \
```

Comment:
I only have no-multilib system to check, but after unpacking the respective ebuilds
`$ find /var/tmp/portage/dev-libs/libfm-qt-9999/work/libfm-qt-9999/ -name CMakeLists.txt | grep -i destination`
and
`$ find /var/tmp/portage/x11-misc/pcmanfm-qt-9999/work/pcmanfm-qt-9999/ -name CMakeLists.txt | grep -i destination`
don't return anyting.